tests/monitors/kubernetes_cluster/kubernetes_cluster_test.py

```diff
@@ -309,6 +309,15 @@ def test_containers(k8s_cluster):
                     container_id = container_status.container_id.replace("docker://", "").replace("cri-o://", "")
                     containers_cache[container_status.name] = container_id
                     assert wait_for(p(has_datapoint, agent.fake_services, dimensions={"container_id": container_id}))
+                    assert wait_for(
+                        p(
+                            has_dim_prop,
```

Comment:
Be more specific here on the expected value of `container_status`.  Would be nice to test for the terminated/waiting states too but not sure how easy it is to force a container into that state when managed by Kubernetes.  If it isn't too convoluted to do, test for those other states as well.

Reply:
Updated the tests to pick up container status from the container objects retrieved. Let me know what you think.

Reply:
Seems good.